utils/validations/forms.ts

```diff
@@ -60,6 +61,11 @@ const validNonZeroNumber = {
     message: 'Rating must be at least 1 star.'
 }
 
+const validHttpOrHttpsUrl = {
```

Comment:
Naming update 🙃 
`validWebUrl` instead of `validHttpOrHttpsUrl`. Gets the point across without all the wordiness